Dockerfile

```diff
@@ -28,6 +31,9 @@ ADD X11vnc.run /etc/service/X11vnc/run
 ADD noVNC.run /etc/service/noVNC/run
 ADD df.run /etc/service/df/run
 
+#
+RUN rm /df_linux/libs/libstdc++.so.6 /df_linux/libs/libgcc_s.so.1
```

Comment:
> "If you see any errors related to libstdc++ when starting DF (for example, "version `CXXABI_1.3.8' not found"), try removing libstdc++.so.6 and libgcc_s.so.1 from the "libs" folder. This will cause DF to fall back to your system versions of these libraries, which are more up-to-date on many modern systems than the libraries that DF distributes."


https://bay12games.com/dwarves/mantisbt/view.php?id=11681